### DISCUSSION TITLE
Decode byte

### DISCUSSION THREAD

**Discussion by OliverTrust** [SteamRE/SteamKit#1440] [General]
Is there any way to decode the response of this protobuf message?
https://github.com/SteamDatabase/Protobufs/blob/57781c1d10d35910fe3d1918b3d1e429e6488c37/csgo/base_gcmessages.proto#L271

![1](https://github.com/user-attachments/assets/8b3a8ba1-c82b-4c26-a641-3115d52b9ebd)

I don't know what i must do next to get clean and readable response

**Comment by SeRi0uS007** [comment]
In my latest experiments, the price sheet was a Valve Key-Value string compressed by the LZMA algorithm.

Very likely you will have to parse and read this particular dump-file yourself. NetHookAnalyzer is not suitable in this case.

**Reply by yaakov-h** [reply]
See https://github.com/SteamRE/SteamKit/pull/1441 for getting NHA to do this.

**Reply by OliverTrust** [reply]
AWESOME, THANK YOU!
